Graphics/GraphicsEngineD3D12/include/pch.h

```diff
@@ -58,6 +58,10 @@ enum D3D12_SHADING_RATE_COMBINER
 {
 };
 
+#ifndef NTDDI_WIN10_CO // First defined in Win SDK 10.0.22000.0
+#    define D3D_FEATURE_LEVEL_12_2 0xc200
```

Comment:
Please define it as `constexpr` constant of type `D3D_FEATURE_LEVEL` similar to `D3D12_RESOURCE_STATE_SHADING_RATE_SOURCE` below.

Also, please move this above the 10.0.19041.0 SDK to keep the defines ordered by versions.